arklib/Cargo.toml

```diff
@@ -8,7 +8,7 @@ name = "arklib"
 crate-type = ["cdylib", "staticlib"]
 
 [dependencies]
-arklib = { git = "https://github.com/ARK-Builders/arklib", rev = "b743977" }
+arklib = { git = "https://github.com/hhio618/arklib"}
```

Comment:
_As usually, we should not forget to fix it._

Reply:
Sure, it's just for passing the GitHub actions for this PR.
I'll definitely remove them as the last step.

Reply:
Yep, I know, no worries :) I'm putting this reminder mostly for me, in order to check before merge.

Reply:
But regarding GitHub Actions — I think it would be also enabled from branch, if not then we would need to fix it this way.